packages/cms/pages/settings/roles-and-permissions/roles-and-permissions.tsx

```diff
@@ -337,13 +363,20 @@ const RolesFlyout: React.FC<{
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiButton
-              onClick={() => {
+              onClick={async () => {
+                loading = true
                 submit?.()
+                if (isEditing)
```

Comment:
More readable to use the full `{}` on if / else statements.

Reply:
okay